lerna.json

```diff
@@ -0,0 +1,7 @@
+{
+  "lerna": "2.0.0-rc.5",
+  "packages": [
+    "packages/*"
+  ],
+  "version": "0.0.0"
+}
```

Comment:
Yes, I'll add an `.editorconfig` shortly, WordPress is switching back to tabs for `.json` files:

See  [#WP40946](https://core.trac.wordpress.org/ticket/40946) and https://github.com/WordPress/gutenberg/pull/1067